datanode/sqlstore/migrations/0049_maker_fees_in_fees_stats.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+
+alter table fees_stats
+    add column total_maker_fees_received jsonb not null default '[]';
```

Comment:
I don't really think we should be setting a default for a field that is `not null`, we should enforce it